Scala/Scala.sublime-syntax

```diff
@@ -147,6 +147,46 @@ contexts:
     - match: '{{typeprefix}}'
       push: single-type-expression
 
+  annotation:
+    - match: '(@){{plainid}}\.'
+      captures:
+        0: storage.modifier.annotation.scala
+        1: punctuation.definition.annotation.scala
+      push:
+        - meta_scope: meta.annotation.scala
+        - match: '{{plainid}}\.'
+          scope: storage.modifier.annotation.scala
+        - match: '{{plainid}}'
+          scope: storage.modifier.annotation.scala
+          set: annotation-parameters
+    - match: '(@){{plainid}}'
+      captures:
+        0: storage.modifier.annotation.scala
+        1: punctuation.definition.annotation.scala
+      push: annotation-parameters
+
+  annotation-parameters:
+    - meta_scope: meta.annotation.scala
+    - match: \(
+      scope: punctuation.definition.arguments.annotation.begin.scala
+      push:
+        - match: ','
+          scope: punctuation.separator.arguments.annotation.scala
+        - match: \)
+          scope: punctuation.definition.arguments.annotation.end.scala
+          pop: true
+        - include: main
+    - match: \[
+      scope: punctuation.definition.arguments.annotation.begin.scala
+      push:
+        - match: \]
+          scope: punctuation.definition.arguments.annotation.end.scala
+          pop: true
+        - include: delimited-type-expression
+    - match: \s*
```

Comment:
I would probably move this as part of the next negative lookahead, e.g. `(?![(\[\s])`